pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -149,6 +151,9 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 			"any number of times to set multiple annotations. "+
 			"To unset, specify the annotation name followed by a \"-\" (e.g., name-).")
 	p.markFlagMakesRevision("annotation")
+
+	command.Flags().IntVar(&p.ScaleInit, "scale-init", 0, "Initial scale for a service")
```

Comment:
Cool that default is 0 but need to reflect this in the generated docs for the help output.

Reply:
I will denote it in the description message.

Reply:
It is indeed "1". The default "0" here in client is actually of no significance and could be any value, because it's not like 0 is going to be sent as initial Scale value if the flag `--scale-init` is not passed and in which case, the server will set 1 as default. In case you explicitly specify this option with a value, that value will be validated in the server.

Reply:
yea, the value should be processed only if the flag `Changed`, so the default value is irrelevant here